src/app/static/src/app/store/modelOptions/optionsUtils.ts

```diff
@@ -21,10 +30,27 @@ function hasOptions(control: Control): control is ControlWithOptions {
     return control.type === "select" || control.type === "multiselect";
 }
 
+function isNumberControl(control: Control): control is NumberControl {
+    return control.type == "number"
+}
+
+function checkNumberControlValid(control: NumberControl): boolean {
+    let valid = true;
+
+    if (control.min !== undefined && typeof control.value === "number") {
+        valid = valid && control.value > control.min
+    }
+    if (control.max !== undefined && typeof control.value === "number") {
+        valid = valid && control.value < control.max
```

Comment:
Min and max are inclusive I think
```suggestion
        valid = valid && control.value >= control.min
    }
    if (control.max !== undefined && typeof control.value === "number") {
        valid = valid && control.value <= control.max
```

Reply:
Good spot! Thanks